kong/plugins/zipkin/opentracing.lua

```diff
@@ -7,15 +7,15 @@ A plugin that derives this should:
   - Implement a :initialise_request(conf, ctx) method if it needs to do per-request initialisation
 ]]
 
-local BasePlugin = require "kong.plugins.base_plugin"
 local subsystem = ngx.config.subsystem
 
-local OpenTracingHandler = BasePlugin:extend()
-OpenTracingHandler.VERSION = "scm"
+local OpenTracingHandler = {
+  VERSION = "scm",
+  -- We want to run first so that timestamps taken are at start of the phase
+  -- also so that other plugins might be able to use our structures
+  PRIORITY = 100000,
+}
 
--- We want to run first so that timestamps taken are at start of the phase
--- also so that other plugins might be able to use our structures
-OpenTracingHandler.PRIORITY = 100000
 
 function OpenTracingHandler:new(name)
   OpenTracingHandler.super.new(self, name or "opentracing")
```

Comment:
Another indicator that it's never executed is the fact that without BasePlugin there's no `.super`, so if this was running, it would have crashed.